Add desktop version

diarisdiakite opened the pull request:
In this PR, I:
- created the branch add_desktop_version and rebased on main
- Added the _media queries_ for min-width: 375 and min-width: 768
- Added a navbar which is displayed only on desktop version
- Added the specific fields related to desktop version to the contact-form
-  Fixed the css issues

shella12 commented on css/styles.css:
```diff
@@ -665,4 +700,28 @@ Contains email and socials
+  .mobile-fields {
+    display: none;
+  }
+
+  input[class="desktop-fields"] {
+    width: 49%;
+    display: inline-flex;
+  }
+
+  form input:first-child {
+    background-color: transparent;
+  }
+
+  .form-button {
+    margin-left: 0;
+    width: 20%; /* TO FIX */
+    height: 100%;
+  }
```
- [ ] As we are aiming for a design identicle to the figma design please 🙏 make sure the paddings in the input and button of your design also match with that of the figma. I suggest adding some padding top and reduce padding from the left of the `message` field. Also please reduce some padding top and bottom of the `name` and `email` field. And add padding top and bottom on the button. 👍 

Figma Design:
<img width="341" alt="Screenshot 2023-03-31 at 9 28 16 AM" src="https://user-images.githubusercontent.com/44798044/229027162-22681766-403b-4e2f-b5d0-c4130bb2aef8.png">

Your design:

<img width="750" alt="Screenshot 2023-03-31 at 9 28 20 AM" src="https://user-images.githubusercontent.com/44798044/229028139-92be76d7-8b79-4246-bdbb-151df1c1d0ce.png">

diarisdiakite replied:
Done

shella12 commented on index.html:
```diff
@@ -10,54 +10,62 @@
+    <section class="headline">
 
-  <section class="headline">
+      <div class="headline-text">
+        <h1>
+          Hey there.
+          <br>
+          I’m Didy.
+          <br>
+          <span class="green">I’m a Full Stack developer from Microverse</span>
+        </h1>
```
- [ ] Please make sure your `main heading `looks just like the one in the figam. I suggest increasing `font-weight `and removing `br ` element from the heading in the desktop version. You can also divide the main heading in to multiple heading tags i.e. `hx` instead of using span in one `h1`

Figma design:

<img width="346" alt="Screenshot 2023-03-31 at 9 26 49 AM" src="https://user-images.githubusercontent.com/44798044/229028704-04fe8b25-1c15-4353-936d-9c2ab5fefe2b.png">

Your design:

<img width="958" alt="Screenshot 2023-03-31 at 9 26 55 AM" src="https://user-images.githubusercontent.com/44798044/229028666-80ab00ff-53b8-4fe6-9684-83e90d8f2432.png">

shella12 commented on css/styles.css:
```diff
@@ -111,21 +110,41 @@ header {
   align-self: flex-end;
 }
 
+.navbar {
+  list-style-type: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+}
+
+.navbar a {
+  text-decoration: none;
+  color: var(--my-text-color);
+  padding: 1% 5%;
+}
+
```
- [ ] Please make sure that mobile design still looks like expected after adding the desktop version as we are aiming for a responsive website so adding one shouldn't effect the other design

Your design:

<img width="381" alt="Screenshot 2023-03-31 at 9 24 07 AM" src="https://user-images.githubusercontent.com/44798044/229029842-ebc70801-f6cc-4f5b-a610-73eb1e023229.png">


Mobile version Figma design:


<img width="134" alt="Screenshot 2023-03-31 at 9 24 31 AM" src="https://user-images.githubusercontent.com/44798044/229029454-1e860cf5-b413-4fc0-bec9-a2b130fbf022.png">

shella12 commented on index.html:
```diff
@@ -10,54 +10,62 @@
+          don’t hesitate to contact me.
+        </p>
+
+        <ul class="social spacing-default">
+          <li><a href="https://twitter.com/diarisdiakite">
+            <i class="fab fa-twitter"></i>
+          </a></li>
+          <li><a href="https://www.linkedin.com/diarisdiakite/">
+            <i class="fab fa-linkedin-in"></i></li>
+          </a></li>
+          <li><a href="https://www.medium.com/#">
+            <i class="fab fa-medium"></i>
+          </a></li>
+          <li><a href="https://www.github.com/diarisdiakite">
+            <i class="fab fa-github"></i>
+          </a></li>
+          <li><a href="https://www.angellist.com/#">
+            <i class="fab fa-angellist"></i>
+          </a></li>
+        </ul>
+      </div>
```
- [ ] Please add some gap between the social icons on the top section to make your design look identical to the [figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468&t=0dp0MALuxE1qoyAu-0)

figma design

<img width="347" alt="Screenshot 2023-03-31 at 9 27 09 AM" src="https://user-images.githubusercontent.com/44798044/229030303-5d51cdf6-70bd-43ab-8094-f25bee40f8c7.png">

your design:

<img width="708" alt="Screenshot 2023-03-31 at 9 27 16 AM" src="https://user-images.githubusercontent.com/44798044/229030281-c2cd2e2b-af81-4a64-9a15-d8e83e67a966.png">

shella12 commented on css/styles.css:
```diff
@@ -561,47 +574,63 @@ Contains email and socials
+    width: 92%;
+  }
+
+  .recent-work-text,
+  .about-myself-text {
+    margin: 0 auto 5% auto;
```
- [ ] The `My Recent Work` heading in the figma design template is `Capatalized` please make sure that your heading is `Capatalized`

Figma design:

<img width="502" alt="Screenshot 2023-03-31 at 9 27 29 AM" src="https://user-images.githubusercontent.com/44798044/229030681-5fa1a8d2-7864-4958-b0d3-9d9941fff184.png">

Your design:

<img width="655" alt="Screenshot 2023-03-31 at 9 27 38 AM" src="https://user-images.githubusercontent.com/44798044/229030719-0092d044-d541-45fa-8ae3-be86026d3b82.png">

shella12 commented on css/styles.css:
```diff
@@ -561,47 +574,63 @@ Contains email and socials
-      position: relative;
-      margin: 1% auto 0% auto;
-    }
+  .recent-work-text img {
+    width: auto;
+    position: relative;
+    margin: 1% auto 0% auto;
+  }
```
- [ ] In the figma design there are suppose to be six project cards. Please make sure your design also includes six cards. 

Your design:

<img width="1499" alt="Screenshot 2023-03-31 at 9 25 46 AM" src="https://user-images.githubusercontent.com/44798044/229031084-39927540-a393-4ab3-ab02-d1277abf5ce0.png">

Figma design:

<img width="500" alt="Screenshot 2023-03-31 at 9 27 34 AM" src="https://user-images.githubusercontent.com/44798044/229031036-0765a456-0fbd-4e2e-b442-47f5d043034b.png">

diarisdiakite replied:
Done

shella12 reviewed (changes requested):
Hi @diarisdiakite, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media1.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Highlights
- All linters passed ✅ 
- Professional README.md ✅   
- Correct use of git flow ✅ 
- Descriptive pull request ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

diarisdiakite commented on css/styles.css:
```diff
@@ -665,4 +700,28 @@ Contains email and socials
     gap: 1%;
   }
+
+  .mobile-fields {
+    display: none;
+  }
+
+  input[class="desktop-fields"] {
+    width: 49%;
+    display: inline-flex;
+  }
+
+  form input:first-child {
+    background-color: transparent;
+  }
+
+  .form-button {
+    margin-left: 0;
+    width: 20%; /* TO FIX */
+    height: 100%;
+  }
```
Done

diarisdiakite commented on css/styles.css:
```diff
@@ -665,4 +700,28 @@ Contains email and socials
+    height: 100%;
+  }
+
+  footer {
+    background: url(../images/mask-group-footer.png);
+    background-repeat: no-repeat;
```
Done

diarisdiakite commented:
In this milestone, I: 
- Added the 6th project Card
- Capitalized the recent works section title
- Corrected the form to keep the one _Full Name_ field for the desktop version according to the requirements
- Fixed the navbar display issue to display only in desktop version and display the toogle icon for the mobile version
- Removed the email _p_ tag from desktop version
- Rearranged the desktop version social icons according to the Figma template
- Adjusted the footer social icons section to be at the bottom of the page
- Added Background images according to the Figma Template

didierganthier reviewed (changes requested):
Hi @diarisdiakite ,

🎉 Great job on addressing the feedback from the previous review! You've made some solid progress in implementing the requested changes. However, there are still a few areas that need some attention before we can move on to the next project.

<img src="https://media.giphy.com/media/e56n4Z0mmkxZS/giphy.gif" />

## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- [ ] Well done overall 💯 but something seems to be wrong with this section, this is how it appears on full-screen 👇🏽 

![Screenshot 2023-03-31 at 23 12 17](https://user-images.githubusercontent.com/45739963/229263026-7516b75f-45b1-43f8-97cf-2fc0d600d226.png)

Make sure to adjust it to match the Figma 🎨 


- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

diarisdiakite commented:
In this milestone, I:
- Added the background images for _About myself_ section with pseudo selectors.
- Rearranged the Css to delete duplicate fields

Lizdev-05 reviewed (approved):
Hi @diarisdiakite, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
<img src="https://media1.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif"/>
## Highlights
All linters passed ✅
Professional README.md ✅
Correct use of git flow ✅
Descriptive pull request ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
